src/main/java/com/mapr/synth/Synth.java

```diff
@@ -337,12 +368,54 @@ public ThreadReport() {
         }
     }
 
-
     static Joiner withCommas = Joiner.on(",");
     static Joiner withTabs = Joiner.on("\t");
 
+    private static void formatTemplate(Format format, Quote quoteConvention, List<String> names, JsonNode fields, Template temp, PrintWriter writer) {
+        try {
+            temp.process(fields, writer);
+        } catch (TemplateException e) {
+            e.printStackTrace();
```

Comment:
Never do this.  If you catch an exception, you have to do something about it.  If you can't do something, then you shouldn't catch it.